.cloudbuild/library_generation/cloudbuild-library-generation-push.yaml

```diff
@@ -15,7 +15,7 @@
 timeout: 7200s # 2 hours
 substitutions:
   _IMAGE_NAME: "gcr.io/cloud-devrel-public-resources/java-library-generation"
-  _GAPIC_GENERATOR_JAVA_VERSION: '2.41.1-SNAPSHOT' # {x-version-update:gapic-generator-java:current}
+  _GAPIC_GENERATOR_JAVA_VERSION: '2.42.0' # {x-version-update:gapic-generator-java:current}
```

Comment:
I already merged the snapshot pr, do we need to change the current version to snapshot?

Reply:
That would the followup PR. If we change it now, we would not get a docker image with 2.42.0 version.